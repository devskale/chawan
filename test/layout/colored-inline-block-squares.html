<div>one
<div style="display: inline-block; margin: 2ch; width: 20ch; height: 20ch; background-color: blue">
?<br>
</div>
<div style="display: inline-block; margin: 6ch; width: 20ch; height: 20ch; background-color: blue">
?<br>
</div>
??
<div style="display: inline-block; margin-left: 5ch; margin-top: 1em; width: 20ch; background-color: green">?<br><br><br><br><br><br><br><br><br><br><br></div>w
<br>
<div style="display: inline-block; margin-left: 6ch; width: 20ch; height: 20ch; background-color: red">?</div>
w two
</div>
