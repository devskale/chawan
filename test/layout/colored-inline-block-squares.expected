


one   [48;2;0;0;255m?                   [49m         [48;2;0;0;255m?                   [49m       ??
      [48;2;0;0;255m                    [49m         [48;2;0;0;255m                    [49m
      [48;2;0;0;255m                    [49m         [48;2;0;0;255m                    [49m
      [48;2;0;0;255m                    [49m         [48;2;0;0;255m                    [49m
      [48;2;0;0;255m                    [49m         [48;2;0;0;255m                    [49m
      [48;2;0;0;255m                    [49m         [48;2;0;0;255m                    [49m
      [48;2;0;0;255m                    [49m         [48;2;0;0;255m                    [49m
      [48;2;0;0;255m                    [49m         [48;2;0;0;255m                    [49m
      [48;2;0;0;255m                    [49m         [48;2;0;0;255m                    [49m
      [48;2;0;0;255m                    [49m         [48;2;0;0;255m                    [49m




     [48;2;0;128;0m?                   [49m
     [48;2;0;128;0m                    [49m
     [48;2;0;128;0m                    [49m
     [48;2;0;128;0m                    [49m
     [48;2;0;128;0m                    [49m
     [48;2;0;128;0m                    [49m
     [48;2;0;128;0m                    [49m
     [48;2;0;128;0m                    [49m
     [48;2;0;128;0m                    [49m
     [48;2;0;128;0m                    [49m
     [48;2;0;128;0m                    [49mw
      [48;2;255;0;0m?                   [49m w two
      [48;2;255;0;0m                    [49m
      [48;2;255;0;0m                    [49m
      [48;2;255;0;0m                    [49m
      [48;2;255;0;0m                    [49m
      [48;2;255;0;0m                    [49m
      [48;2;255;0;0m                    [49m
      [48;2;255;0;0m                    [49m
      [48;2;255;0;0m                    [49m
      [48;2;255;0;0m                    [49m
